src/styles/1-helpers/_mixins.scss

```diff
@@ -0,0 +1,44 @@
+// centering
```

Comment:
This looks much cleaner! 

So people working on different pages should try to re-use variables from the `src/styles` folder right? 

Wondering if there is any particular reason to include the number in the folder paths? The enforced ordering and numeric `1-helpers` rather than `helpers` might create some confusion as we merge or remove some folders? 
But this is quite minor tho.

Reply:
With how it's set up now, we won't have to import the styling files since all of them are being imported in main.scss which then is imported in index.js. This allows us to directly implement styling in our jsx files without imports.

The numbers are just a way to keep the folders organized and also help in ordering styling (since files in 1-helpers have to be imported before 2-5).
![image](https://user-images.githubusercontent.com/66650721/157597880-451db2d7-5efd-43c0-8222-65661b99c6aa.png)

You are right though, adding/removing files can create some issues, but it wouldn't be too difficult to solve since all the imports are handled in main.scss.